frontend/src/components/QuestionsPage/QuestionsWrapper.tsx

```diff
@@ -48,6 +50,8 @@ const WrappedComponent = <P extends object>(
         this.getQueryQuestions(searchQuery);
```

Comment:
Would it be better to use switch/case instead of else/if?

Reply:
If/else is much faster ;)